manifest/manifest.go

```diff
@@ -127,6 +127,12 @@ func (m Manifest) Validate() []error {
 				errors = append(errors, fmt.Errorf("%s links to service: %s which does not expose any ports", entry.Name, l))
 			}
 		}
+
+		// test mem_limit: Docker requires a mem_limit of at least 4mb (or 0)
+		mem_min := Memory(4194304)
+		if entry.Memory < mem_min && entry.Memory != 0 {
+			errors = append(errors, fmt.Errorf("%s has invalid mem_limit %#v: should be either 0, or at least %#v", entry.Name, entry.Memory, mem_min))
```

Comment:
I forgot to mention in my initial pass, this error should state explicitly what's going on.

For example entry.Memory while in bytes, should be displayed in "human readable" format in the error.
`mem := units.HumanSize(float64(entry.Memory))`
And while technically true, do we need to mention 0 as a valid value?

`At least 4MB is required for web.  Found:  "2.097 MB"`
Something along those lines.